p2p/pex/pex_reactor.go

```diff
@@ -464,6 +474,10 @@ func (r *PEXReactor) ensurePeers() {
 		if r.Switch.IsDialingOrExistingAddress(try) {
 			continue
 		}
+		if r.Switch.IsUnconditionalPeer(try.ID) {
+			numToDial++
```

Comment:
so every attempt to dial unconditional peer will result in increasing numToDial? seems wrong to me

Reply:
Oh, you are right, I'll remove this line